webgrid/__init__.py

```diff
@@ -113,7 +113,7 @@ def _assign_to_grid(self):
         grid_cls_cols = grid_locals.setdefault('__cls_cols__', [])
         grid_cls_cols.append(self)
 
-    def __init__(self, label, key=None, filter=None, can_sort=True,
+    def __init__(self, label, key=None, filter=None, can_sort=True, group=None,
```

Comment:
Inserting a new argument into the sequence will have implications for existing column classes. You've trapped the webgrid internally-defined cases, but applications often have their own columns.

Next step here would be to pull this up with one or both of our larger apps, and see if breakage could be resolved into a more backwards-compatible solution. FWIW, the column instantiation process here is clunky and not very extensible overall, but I'd like to try to leave breaking changes to webgrid 2.0.